examples/account_merge.py

```diff
@@ -13,7 +14,8 @@
 # The following source key was created by the friendbot at https://laboratory.stellar.org/#account-creator?network=test
 # before running this example create a new account, fund it and then copy and paste the
 # secret key where the current key is.
-source_secret_key = "SC7AUS23UKVZQL5KMIK4ZK3EZJUS6ZVMTQSVLH3VIK42W6RBQAQXOVQX"
```

Comment:
> Sorry for my late reply, I have been a little busy lately.
> 
> I have an idea, but I'm not sure if this is better.
> 
> After the user downloads these examples, it contains an init script, after executing this script, it will replace the public keys and secrets in each example, and complete the necessary account activation and asset preparation in advance. This keeps the simplicity of each example at the same time.

am just seeing this, my apologies. i think this can work too

Reply:
Yes, I think [cookiecutter](https://github.com/cookiecutter/cookiecutter) is a great implementation.

Reply:
Great, I will take a look and revert back